examples/acquiring_data/downloading_hmi.py

```diff
@@ -32,15 +36,28 @@
 # don't provide a path it will download the file into the sunpy data directory.
 # The output provides the path of the downloaded files.
 # The result can be from several data clients, so we have to index the client first
-# client and then index the file.
+# and then index the file.
 
 # Slice the first record returned by the first client.
-downloaded_file = Fido.fetch(result[0, 0])
+downloaded_file = Fido.fetch(result[0])
 print(downloaded_file)
 
 ###############################################################################
-# Now load it into a map and plot it
+# Now load it into a map and plot it.
+# We see that solar North is pointed down instead of up in this image, which is
+# indicated by the coordinates (that range from positive to negative, rather
+# than negative to positive).
+# We can verify the rotation of the image by querying the keyword `CROTA2`,
+# which indicates the number degrees the image must be rotated for solar North
+# to point up.
 hmi_map = sunpy.map.Map(downloaded_file[0])
 fig = plt.figure()
 hmi_map.plot()
 plt.show()
+
+###############################################################################
+# Rotate the image such that solar North is pointed up. The map automatically
+# incorporates the value of `CROTA2` into the rotation.
+hmi_rotated = hmi_map.rotate(order=3)
```

Comment:
You need to explain why you're choosing to use `order=3` (or a reader might interpret that `order=3` is a required keyword argument to utilize the `CROTA2` value).

Reply:
👍 I will add this.

Reply:
What is the reason? It's just faster?

Reply:
It's because `order=3` doesn't convert all of your NaNs to zeros. If you use the default (`order=4`), you can't see the edge of the disk.